R/audits.R

```diff
@@ -114,7 +53,7 @@ audit_r_versions <- function(apps) {
 
   # timeline
   p2 <- ggplot2::ggplot(timeline) +
-    ggplot2::geom_point(pch = 4, ggplot2::aes(x = published, color = r_version, y = r_version)) +
+    ggplot2::geom_point(pch = 4, ggplot2::aes(x = last_deployed_time, color = r_version, y = r_version)) +
```

Comment:
Should we add `last_deployed_time` to the global vars list? Or does `vars("last_deployed_time")` work here?

Reply:
Added to the list